SharedCode/gradle-config.gradle

```diff
@@ -29,8 +29,15 @@ task secretKeysSetup {
             segmentWearableWriteKey = "THE_SEGMENT_API_WRITE_KEY_WEARABLE_DEMO_GOES_HERE"
         }
 
+        String mapboxAccessToken = "$System.env.MAPBOX_ACCESS_TOKEN"
+        if (mapboxAccessToken == "null") {
+            System.out.println("You should set the MAPBOX_ACCESS_TOKEN environment variable.")
+            mapboxAccessToken = "YOUR_MAPBOX_ACCESS_TOKEN_GOES_HERE"
+        }
+
         String tokenFileContents = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n" +
                 "<resources>\n" +
+                "    <string name=\"access_token\">" + mapboxAccessToken + "</string>\n" +
```

Comment:
translatable should be set to false right?

Reply:
Correct. Fixed.